releases/3.4.0/artifacts.json

```diff
@@ -0,0 +1,60 @@
+{
+  "3.4.0": {
+    "holoscan": {
+      "debian-version": "3.4.0.2-1",
+      "wheel-version": "3.4.0",
+      "base-images": {
+        "dgpu": "nvcr.io/nvidia/cuda:12.8.1-runtime-ubuntu24.04",
+        "igpu": "nvcr.io/nvidia/tensorrt:25.03-py3-igpu"
+      },
+      "build-images": {
+        "igpu": {
+          "jetson-agx-orin-devkit": "nvcr.io/nvstaging/holoscan/holoscan:v3.4.0.2-igpu",
+          "igx-orin-devkit": "nvcr.io/nvstaging/holoscan/holoscan:v3.4.0.2-igpu"
```

Comment:
Note rc versions != final release

Reply:
Yep! I will update this again when final artifacts become available.